README.md

```diff
@@ -2,20 +2,20 @@
 
 A set of Python Notebooks to demonstrate how to access the images and metadata from the Image Data Repository (IDR, https://idr.openmicroscopy.org), including features and all descriptive tags.
 
-Full access to IDR metadata and images is provided through the standard OMERO API, documentation for which can be found [here](https://www.openmicroscopy.org/site/support/omero5.2/developers/), with the Python bindings found specifically [here](https://www.openmicroscopy.org/site/support/omero5.2/developers/Python.html). The notebooks in this repository are meant to exemplify the use of that API in the context of the IDR, and the sort of queries that can be done. In particular, they show how to reproduce Figure 1 and Figure 2 of the paper.<sup>[1](#footnote1)</sup> They also make use of the [scipy](https://www.scipy.org/) ecosystem, including [pandas](http://pandas.pydata.org).
+Full access to IDR metadata and images is provided through the standard OMERO API, documentation for which can be found [here](https://docs.openmicroscopy.org/omero/5.4.5/developers/index.html), with the Python bindings found specifically [here](https://docs.openmicroscopy.org/omero/5.4.5/developers/Python.html). The notebooks in this repository are meant to exemplify the use of that API in the context of the IDR, and the sort of queries that can be done. In particular, they show how to reproduce Figure 1 and Figure 2 of the paper.<sup>[1](#footnote1)</sup> They also make use of the [scipy](https://www.scipy.org/) ecosystem, including [pandas](http://pandas.pydata.org).
```

Comment:
to point to the latest version of omero you should use
https://docs.openmicroscopy.org/latest/omero/developers/index.html
otherwise this will need to be updated when 5.4.6 is out

Reply:
I went for `latest/omero5.4/` since the IDR may lag behind.